tethys_cli/manage_commands.py

```diff
@@ -35,10 +38,13 @@ def add_manage_parser(subparsers):
     manage_parser.add_argument('-l', '--link', required=False, action='store_true',
                                help='Only used with collectstatic command. Link static directory to STATIC_ROOT '
                                     'instead of copying it. Not recommended.')
+    manage_parser.add_argument('--django-help', action='store_true',
+                               help='Display the help for specific scommands coming from Django. '
+                               'E.g. "tethys manage <django command> --django-help"')
     manage_parser.set_defaults(func=manage_command)
 
 
-def manage_command(args):
+def manage_command(args, django_args=""):
```

Comment:
"django_args" -> "unknown_args" for consistency.

Reply:
done